libpam_internal/pam_debug.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * This provides the necessary functions to do debugging in PAM.
+ * Cristian Gafton <gafton@redhat.com>
+ */
+
+#include "config.h"
+
+#ifdef PAM_DEBUG
+
+#define PAM_DEBUG_C 1
+
+#include "security/_pam_macros.h"
+
+#undef PAM_MACROS_H
+#undef PAM_NO_HEADER_FUNCTIONS
+#include "security/_pam_macros.h"
```

Comment:
Let's move `#define PAM_DEBUG_C 1` after `#undef PAM_NO_HEADER_FUNCTIONS` for better readability.

Reply:
Adjusted